inst/assets/bibliography.bib

```diff
@@ -2244,4 +2244,55 @@ @article{Costea2014
   doi = {https://doi.org/10.1038/nmeth.2897},
   url = {https://www.nature.com/articles/nmeth.2897},
   author = {Costea, Paul I. and Zeller, Georg and Sunagawa, Shinichi and Bork, Peer}
+}
+
+@article{Sanchez2022,
+    author = {Sánchez-Sánchez, Pedro and Santonja, Francisco J and Benítez-Páez, Alfonso},
+    title = "{Assessment of human microbiota stability across longitudinal samples using iteratively growing-partitioned clustering}",
+    journal = {Briefings in Bioinformatics},
+    volume = {23},
+    number = {2},
+    pages = {bbac055},
+    year = {2022},
+    month = {02},
+    issn = {1477-4054},
+    doi = {10.1093/bib/bbac055},
+    url = {https://doi.org/10.1093/bib/bbac055},
```

Comment:
you can remove the URLs when DOI is already provided

Reply:
ACK